src/applications/MachineShop.java

```diff
@@ -39,15 +39,15 @@ static void startShop(SimulationSpecification specification) {
         setUpJobs(specification);
 
         for (int p = 1; p <= numMachines; p++)
-            changeState(p);
+            machineArray[p].changeState();
     }
 
     private static void createEventAndMachineQueues(SimulationSpecification specification) {
         // create event and machine queues
-        eList = new EventList(specification.getNumMachines(), MachineShopSimulator.largeTime);
+        eList = new EventList(specification.getNumMachines(), largeTime);
```

Comment:
I think that this is overall a better place to have time related variables, but the name 'largeTime' could probably have been named something a little better with this change.